### PR TITLE
VPC module

### PR DESCRIPTION
**Pull request by miifeanyi**

### CODE REVIEW

File: example.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.14"
+}
+
+module "my_vpc" {
+  source = "./modules/vpc"
+}
```
Comment (vladimirmukhin) [inline example.tf]: It's a good practice to add newlines at the end of each file.

File: example.tf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.14"
+}
+
+module "my_vpc" {
+  source = "./modules/vpc"
```
Comment (vladimirmukhin) [inline example.tf]: Input values should be explicitly specified.
If you rely on defaults, there would be a high chance of human error when deploying complex infrastructures.

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]
Looks good except for two minor issues.

File: example.tf (hunk 1 above)
Comment (miifeanyi) [reply example.tf]: new line added at end

**Pull request merged**